src/app/students/columns.tsx

```diff
@@ -0,0 +1,50 @@
+"use client"
+
+import { Student } from "@/_temp_types/student"
+import { ColumnDef, Row } from "@tanstack/react-table"
+import { Badge } from "@/components/ui/badge"
+import { DataTableColumnHeader } from "@/components/ui/table-column-header"
+
+type SectionFilterValue = string[]; // Define the type for the filter value
+
+export const columns: ColumnDef<Student>[] = [
+    {
+        id: "firstName",
+        header: ({ column }) => <DataTableColumnHeader column={column} title="First Name" hasDropDownMenu={false}/>,
+        accessorFn: (row) => row.name.split(',')[1], // Assuming the first part is the first name
```

Comment:
eslint giving " 'row.name' is possibly 'undefined'.ts(18048) " as a result of making the name field of Student as optional.